src/server/Modules/EmployeeManagement/Modules.EmployeeManagement/Controllers/StaffController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Modules.EmployeeManagement.Core.Features.Staff.Commands;
+using Shared.Infrastructure.Controllers;
+
+namespace Modules.EmployeeManagement.Controllers
+{
+    [ApiVersion("1")]
+    public class StaffController : BaseController
+    {
+
+        [HttpPost]
+        public async Task<ActionResult> EmployStaffAsync(EmployStaffCommand command)
+        {
+            return StatusCode((int)HttpStatusCode.Created, await Mediator.Send(command));
+        }
+    }
+}
```

Comment:
```suggestion

namespace Modules.EmployeeManagement.Controllers;

[ApiVersion("1")]
public class StaffController : BaseController
{
    [HttpPost]
    public async Task<ActionResult> EmployStaffAsync(EmployStaffCommand command)
        => StatusCode((int)HttpStatusCode.Created, await Mediator.Send(command));
}
```

We could use the simpler syntax here if you wanted - but not important!

Reply:
No logic in the controller, just simplicity ❤️ 🎉